bin/docker_start.sh

```diff
@@ -2,24 +2,16 @@
 
 set -ex
 
-# Wait for the database container
+# wait for required services
 # See: https://docs.docker.com/compose/startup-order/
-export PGHOST=${DB_HOST:-db}
-export PGPORT=${DB_PORT:-5432}
+${SCRIPTPATH}/wait_for_db.sh
```

Comment:
this is at least infrastructure independent - health checks have different definitions in docker-compose/docker without compose/kubernetes and it makes the dependency explicit

I think it may work because the undefined/empty variable points to `/wait_for_db.sh` which likely happens to be exactly the file path where it's copied in the docker image?

Reply:
> likely happens to be exactly the file path where it's copied in the docker image?
correct :)